openslides/mediafiles/static/css/mediafiles/projector.css

```diff
@@ -0,0 +1,27 @@
+img.projector-image {
+    width: 100%;
+}
+
+div.projector-image {
+    width: 100%;
+    height: 100%;
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: 50% 50%;
+}
+
+.video-container {
+	width: 100%;
+    height: 100%;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+
+.video-container > * {
+	flex: 1 1 auto;
+	max-width: 100%;
+	max-height: 100%;
+	width: 100%;
+	height: 100%;
+}
```

Comment:
remove this file and move content to core/static/css/projector.css. Note: Use only 4 spaces indentation.